style/js/shield_defs.js

```diff
@@ -140,8 +140,17 @@ export function loadShields(shieldImages) {
 
   shields["US:US:Truck"] = banneredShield(shields["US:US"], ["TRK"]);
   shields["US:US:Spur"] = banneredShield(shields["US:US"], ["SPUR"]);
+  shields["US:US:Connector"] = banneredShield(shields["US:US"], ["CONN"]);
   shields["US:US:Bypass"] = banneredShield(shields["US:US"], ["BYP"]);
   shields["US:US:Business"] = banneredShield(shields["US:US"], ["BUS"]);
+  shields["US:US:Business:Alternate"] = banneredShield(shields["US:US"], [
+    "BUS",
+    "ALT",
+  ]);
+  shields["US:US:Business:Truck"] = banneredShield(shields["US:US"], [
+    "BUS",
+    "TRK",
+  ]);
```

Comment:
Need to investigate.
The sole example of this tagging in use: https://www.openstreetmap.org/relation/1870734
is in fact Business:Truck
However, the photo of the shield seems to indicate that the ordering should be Truck:Business